packages/py3_unsupported_python.rb

```diff
@@ -1,16 +1,17 @@
 require 'buildsystems/pip'
 
 class Py3_unsupported_python < Pip
-  description 'This package can be used as a conditional dependency to indicate lack of support for particular versions of Python.'
+  version '1.0.0-py3.12'
   homepage 'https://github.com/ntessore/unsupported-python'
   version '1.0.0-py3.12'
   license 'GPL-3+'
   compatibility 'all'
   source_url 'SKIP'
-
-  binary_sha256({})
+  binary_compression 'tar.zst'
 
   depends_on 'python3' => :build
 
+  no_compile_needed
```

Comment:
Adding both a binary_compression value and no_compile_needed seems misleading-- they're essentially conflicting values, after all.